buildman/jobutil/buildjob.py

```diff
@@ -145,6 +145,26 @@ def _determine_cached_tag_by_tag(self):
 
             return None
 
+    def extract_dockerfile_args(self):
+        dockerfile_path = self.build_config.get("build_subdir", "")
+        context = self.build_config.get("context", "")
+        if not (dockerfile_path == "" or context == ""):
+            # This should not happen and can be removed when we centralize validating build_config
```

Comment:
Should a `TODO` be added here, or in a future Pull-Request, related to centralized validation of build_config?